PYME/config.py

```diff
@@ -31,6 +31,11 @@
       |     |
       |     |- recipes
       |           |- anothermodule.txt
+      |     |- reports
+      |     |     |- templates
+      |     |     |     |-somemodule.txt
```

Comment:
Need examples / documentation of what the templates and filters config files will look like

Reply:
Do we want that here, or is it OK to leave it just in the template plugin repository?

Reply:
Now added